Manifest file not closed

It seems that the manifest file is not closed after db.Close(). This can be reproduced using the code below on windows - windows doesn't allow you to delete the folder when it contains an opened file. 

```
package main

import (
  "os"
  "runtime"

  "github.com/cockroachdb/pebble"
)

func main() {
  if runtime.GOOS != "windows" {
    panic("please run on windows")
  }
  dir := "test-data"
  db, err := pebble.Open(dir, &pebble.Options{})
  if err != nil {
    panic(err)
  }
  db.Close()
  if err := os.RemoveAll(dir); err != nil {
    // expected to reach here on windows for pebble v0.0.0-20191101210655-58fbf96a8f51
    panic(err)
  }
}
```
I got the following error message from the last panic()

```
panic: remove test-data\MANIFEST-000001: The process cannot access the file because it is being used by another process.
```

--- a/db.go
+++ b/db.go
@@ -750,6 +750,10 @@ func (d *DB) Close() error {
 	}
 	err = firstError(err, d.fileLock.Close())
 
+	// Note that versionSet.close() only closes the MANIFEST. The versions list
+	// is still valid for the checks below.
+	err = firstError(err, d.mu.versions.close())
+
 	err = firstError(err, d.dataDir.Close())
 	if d.dataDir != d.walDir {
 		err = firstError(err, d.walDir.Close())

--- a/testdata/event_listener
+++ b/testdata/event_listener
@@ -201,5 +201,6 @@ close
 ----
 sync: wal/000013.log
 close: wal/000013.log
+close: db/MANIFEST-000017
 close: db
 close: wal

--- a/version_set.go
+++ b/version_set.go
@@ -246,6 +246,15 @@ func (vs *versionSet) load(dirname string, opts *Options, mu *sync.Mutex) error 
 	return nil
 }
 
+func (vs *versionSet) close() error {
+	if vs.manifestFile != nil {
+		if err := vs.manifestFile.Close(); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 // logAndApply logs the version edit to the manifest, applies the version edit
 // to the current version, and installs the new version. DB.mu must be held
 // when calling this method and will be released temporarily while performing

